packages/edit-site/src/components/canvas-loader/style.scss

```diff
@@ -16,6 +16,10 @@
 	& > div {
```

Comment:
What is this style used for? Perhaps for the progress bar? IF not, couldn't we reuse it for the progress bar?